app/navigation/ReturnsJourneyNavigator.scala

```diff
@@ -107,9 +107,10 @@ class ReturnsJourneyNavigator @Inject()(
     controllers.returns.credits.routes.ConfirmPackagingCreditController.onPageLoad(mode)
 
   def confirmCreditRoute(mode: Mode): Call =
-    if (mode.equals(CheckMode))
-      routes.ReturnsCheckYourAnswersController.onPageLoad()
-    else
+    // TODO disable for now - otherwise can try to jump to full CYA on first pass through journey -> crash
```

Comment:
think we have an idea of how to solve this, I can impliment today?

Reply:
```
def trueCreditsCheckMode(userAnswers: UserAnswers, mode: Mode) = {
    mode == CheckMode &&
    userAnswers.get(ManufacturedPlasticPackagingWeightPage).isDefined &&
    userAnswers.get(ImportedPlasticPackagingWeightPage).isDefined &&
    userAnswers.get(DirectlyExportedWeightPage).isDefined &&
    userAnswers.get(AnotherBusinessExportedWeightPage).isDefined &&
    userAnswers.get(NonExportedHumanMedicinesPlasticPackagingWeightPage).isDefined &&
    userAnswers.get(NonExportedRecycledPlasticPackagingWeightPage).isDefined
  }
```

Reply:
Cool, thanks. New branch tho?